### PR TITLE
Add info for Microsoft Teams webhooks

### PR DESCRIPTION
**Pull request by komalali**
## Description

Add info for Microsoft Teams webhooks

## Checklist:

- [ ] I have reviewed the [style guide](https://github.com/pulumi/pulumi-hugo/blob/master/STYLE-GUIDE.md).
- [ ] If blogging, I have reviewed the [blogging guide](https://github.com/pulumi/pulumi-hugo/blob/master/BLOGGING.md).
- [ ] I have manually confirmed that all new links work.
- [ ] I added aliases (i.e., redirects) for all filename changes.
- [ ] If making css changes, I rebuilt the bundle.

### CODE REVIEW

File: themes/default/content/docs/pulumi-cloud/webhooks.md
Comment (kimberleyamackenzie) [inline themes/default/content/docs/pulumi-cloud/webhooks.md:131]: This new syntax feels off to me with dropping the `-formatted`?  But words 🤷‍♀️ - will leave it up to you.

File: themes/default/content/docs/pulumi-cloud/webhooks.md
Comment (kimberleyamackenzie) [inline themes/default/content/docs/pulumi-cloud/webhooks.md:198]: 🎉 nice catch

**Review by kimberleyamackenzie — APPROVED** [review]
lgtm!

File: themes/default/content/docs/pulumi-cloud/webhooks.md
Comment (cnunciato) [reply themes/default/content/docs/pulumi-cloud/webhooks.md:131]: The original text seems a bit off too -- "select between ... a webhooks". 😄 

I can't submit the suggestion directly because GitHub, but if it helps, here's what I'd suggest:

```
1. Navigate to **Settings** > **Webhooks**.
2. Select **Create webhook**.
3. Under Destination, choose **Webhook**, **Slack**, or **Microsoft Teams**.
    * For generic JSON webhooks, provide a display name, payload URL, and optional validation secret.
    * For Microsoft Teams webhooks, provide a display name and Microsoft Teams webhook URL.
	* For Slack webhooks, provide a display name and Slack webhook URL.
4. Choose between receiving all events or only specific events using the filters menu.
```

If this looks okay to you also, would you mind applying to the Stack Webhook section also (and adjust accordingly for Deployments)?

File: themes/default/content/docs/pulumi-cloud/webhooks.md
Comment (cnunciato) [inline themes/default/content/docs/pulumi-cloud/webhooks.md:181]: The "formatted" part seems like an implementation detail. I'd probably just mirror what we call them generally and in the UI.

```suggestion
Slack Webhooks allow you to seamlessly integrate notifications about your Pulumi stacks and organizations
```

File: themes/default/content/docs/pulumi-cloud/webhooks.md
Comment (cnunciato) [inline themes/default/content/docs/pulumi-cloud/webhooks.md:192]:
```suggestion
Microsoft Teams Webhooks allow you to seamlessly integrate notifications about your Pulumi stacks and organizations
```

File: themes/default/content/docs/pulumi-cloud/webhooks.md
Comment (cnunciato) [inline themes/default/content/docs/pulumi-cloud/webhooks.md:196]: Are these specific to Pulumi Deployments? If so, I'd probably change this at least to Deployments, if not Pulumi Deployments, to better distinguish them from regular ol' deployment notifications):

```suggestion
### Pulumi Deployments Webhooks
```

I can't see what these look like in the UI, so this is assuming they show up as Pulumi Deployments as a destination. If not feel free to adjust accordingly!

File: themes/default/content/docs/pulumi-cloud/webhooks.md
Comment (cnunciato) [inline themes/default/content/docs/pulumi-cloud/webhooks.md:198]:
```suggestion
The Pulumi Deployments webhook destination lets you trigger updates on other stacks via [Pulumi Deployments](/docs/pulumi-cloud/deployments/), usually in response to `update_succeeded` events. This enables you to keep dependent stacks up to date automatically which is often necessary when using [stack references](/docs/concepts/stack/#stackreferences).
```

File: themes/default/content/docs/pulumi-cloud/webhooks.md
Comment (cnunciato) [inline themes/default/content/docs/pulumi-cloud/webhooks.md:200]:
```suggestion
Deployments webhooks require that your stacks are configured with [Deployment Settings](/docs/pulumi-cloud/deployments/reference/#deployment-settings).
```

File: themes/default/content/docs/pulumi-cloud/webhooks.md
Comment (komalali) [reply themes/default/content/docs/pulumi-cloud/webhooks.md:196]: The UI calls it "Deployment" so will leave this as-is.

File: themes/default/content/docs/pulumi-cloud/webhooks.md
Comment (komalali) [reply themes/default/content/docs/pulumi-cloud/webhooks.md:200]: UI calls this a "Deployment" webhook so leaving as-is.

**Pull request merged**
